templates/patterns/button/button.ui_patterns.yml

```diff
@@ -38,7 +41,12 @@ button:
       type: "text"
       label: "Icon"
       description: "An icon to use with the button."
-      preview: "caret-right"
+      preview: "check"
+    icon_position:
+      type: "text"
+      label: "Icon position"
+      description: "Position of link icon (can be 'before' or 'after') if icon is available"
+      preview: 'before'
```

Comment:
Since this is a new property, let's default it to "after" if empty, in `pattern-button.html.twig`.

Reply:
ok.